check_if_image_tag_exists/main.py

```diff
@@ -33,18 +33,26 @@ def check_if_tag_exists(raw_image_path):
         print tag
 
     if image_tag in existing_tags:
-        sys.exit("Tag already exists in remote repository! Exiting build.")
-    print "Tag does not exist in remote repository! Continuing with build."
+        print "Tag \'" + image_tag + "\' already exists in remote repository!"
```

Comment:
Could you use string formatting here instead of escaping?

`"Tag {0} already exists!".format(image_tag)`